src/modules/Test-SdnExpressBgp.psm1

```diff
@@ -694,25 +694,26 @@ function Test-SdnExpressBGP {
 
             $RestoreMuxState = $false
             $mux = Get-Service -Name 'SlbMux' -ErrorAction SilentlyContinue
-            if ($null -ne $mux) {
+            if ($mux) {
                 $muxstartup = $mux.starttype
                 $muxstatus = $mux.status
 
                 if (($muxstatus -ne "Stopped") -or ($Muxstartup -ne "Disabled")) {
                     if ($force) {
                         $RestoreMuxState = $true
-                        Set-Service -Name -startup Disabled
-                        stop-Service -Name
+                        Set-Service -Name 'SlbMux' -StartupType Disabled
+                        Stop-Service -Name 'SlbMux'
                     }
                     else {
-                        throw "SLB Mux service is active.  Use -force to temporarily disable it during test."
+                        throw "SLB Mux service is active. Use -force to temporarily disable it during test."
                     }
                 }
             }
             else {
-                $muxstate = $null
+                $muxStatus = $null
```

Comment:
Why is assigning this to null needed?

Reply:
@andrwli so this code was inherited into SdnDiagnostics. There was a $muxState, however it was never used anywhere so my assumption was that it was meant to be $muxStatus which is what I updated to.